trin-core/src/types/accumulator.rs

```diff
@@ -44,6 +45,59 @@ impl MasterAccumulator {
         }
     }
 
+    pub fn latest_height(&self) -> u64 {
+        let historical_height = self.epoch_number() * EPOCH_SIZE as u64;
+        historical_height + self.current_epoch.header_records.len() as u64
```

Comment:
Is it possible to overflow here?

Reply:
Imo, not really. Doubt the blockchain height will ever exceed the largest `u64` - or come even remotely close. I can update this to check if you think we should

Reply:
I'm happy to leave it as it is, IMO panicking is good when it indicates a bug in our code.

Reply:
So just to be specific @ogenev -- your suggestion here would be to use something like `u64::try_from(...).unwrap()`, right? Yeah, that totally works for me too.

Reply:
> So just to be specific @ogenev -- your suggestion here would be to use something like `u64::try_from(...).unwrap()`, right? Yeah, that totally works for me too.

Not sure if we need `try_from` and `unwrap`, the code will panic with an overflow error if the sum of both values exceeds the return type of u64.

Reply:
Fair enough. I don't mind reverting the latest commit if you don't think the `expect()` messages would be useful?

Reply:
> Fair enough. I don't mind reverting the latest commit if you don't think the `expect()` messages would be useful?

Don't have any preference really.